### PR TITLE
Custom Surveys for Washington Commons Project

### PR DESCRIPTION
**Pull request by Abby-Wheelis**
We're working with the Washington commons project to implement their custom surveys.

Right now I have the xls and xml files for the surveys they sent over. I have noted some extra options in the demographic survey that might be a mistake, so I need to ask about those and make sure the survey is final before I go through with getting the json files, since those are more labor intensive to generate.

### CODE REVIEW

**Comment by Abby-Wheelis** [comment]
I have tested this out by loading in the emulator:
- saw custom demographics survey instead of default
- trip labels with custom survey
- when survey is filled, the "1 mode, 1 purpose" text will show in the label

File: survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        <label>Vehicle</label>
+        <value>vehicle_alone</value>
+      </item>
+      <item>
+        <label>Transit (bus or train)</label>
+        <value>bus</value>
+      </item>
+      <item>
+        <label>On-demand transit (shuttle service)</label>
+        <value>option_12</value>
```
Comment (shankari) [inline survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml:78]: is this correct?

File: survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      <item>
+        <label>Ridesharing (Uber/Lyft) or taxi</label>
+        <value>ridesharing_uber_lyft_or_similar_service</value>
+      </item>
+    </select>
+    <select ref="/a6dPQCzDbhkycTPXT8os2S/Who_was_in_the_vehic_check_all_that_apply">
+      <label>Who was in the vehicle? (check all that apply)</label>
+      <item>
+        <label>Myself</label>
+        <value>option_4</value>
```
Comment (shankari) [inline survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml:101]: Ditto

**Review by shankari — CHANGES_REQUESTED** [review]
It seems like the value should at least somewhat map the label

File: survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml (hunk 2 above)
Comment (Abby-Wheelis) [reply survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml:101]: For both of these, that is the value given in the file I downloaded from the surveys I was sent (it's what I see in the spreadsheet). I could update the spreadsheet myself or circle back with Rey about sending us an updated file.

Comment (Abby-Wheelis) [reply survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml:101]: There's actually a lot of this in the onboarding survey as well, will having values like "option_4" limit the data analysis capacities for them?

**Comment by Abby-Wheelis** [comment]
I went through the spreadsheets, updated names that were still "option_x" and several that were confusing -- some of the onboarding questions were using the same keys for options in questions that were very different

The names should all be intuitive now, and match the labels they represent

**Comment by shankari** [comment]
This is the first one, so it is OK to be so hands-on. But I do wonder what, if anything, we can do to clean up when we scale. We can't do this for every partnership, I think. Should we just allow them to ahead with the bad option labels, and let them find it in testing? Should we include this in validation?

**Review by shankari — APPROVED** [review]

**Comment by Abby-Wheelis** [comment]
We're going to have to make sure that the path to the surveys is valid, and if a trip survey there is a way to populate the text in the button after the user labels (this was one of the things that took some testing). I think that anything we can do to validate or run it against some phone tests is going to be very helpful. 

We can try and validate that nothing is the "option_X" labels, but that won't catch the ones that were just 'weird' like I found in the spreadsheet. Some of it will probably fall to testing and we can try and warn people -- maybe put some instructions for writing surveys somewhere?

**Pull request merged**